src/react/root.component.js

```diff
@@ -1,24 +1,23 @@
-import React from 'react'
-import { render } from 'react-dom'
-import { Router, hashHistory } from 'react-router'
-import useBasename from 'history/lib/useBasename';
+import React from 'react';
```

Comment:
just added some semicolons and converted some spaces to tabs to match the rest of the file